spec/bolt/executor_spec.rb

```diff
@@ -309,6 +309,28 @@ def mock_node_results(run_as = nil)
     end
   end
 
+  it "traps SIGINT" do
+    skip "hangs on Windows" if Gem.win_platform?
```

Comment:
This test hangs at least in AppVeyor. I'm going to check it outside of AppVeyor tomorrow.

Reply:
The test does work on Windows but when the rspec is executed from a batch script, the batch script prints _Terminate batch job (Y/N)?_ message and waits for user input when the rspec finishes, in effect it hangs.
The problem is that when the SIGINT is sent it is delivered not only to the ruby process executing the rspec tests but rather to the entire process group including the batch script from which the ruby process is executed and that's why the script prints the message and waits for the user input and in effect hangs.